### PR TITLE
Rename update namespace mutation

### PR DESCRIPTION
**Pull request by NetanelBollag**
### Description

Update `namespaces update` command with new API

### CODE REVIEW

**Comment by NetanelBollag** [comment]
@orishoshan do we need to update the command itself to `otterize namespaces associate-to-environment`?

**Comment by orishoshan** [comment]
> @orishoshan do we need to update the command itself to `otterize namespaces associate-to-environment`?

Yes please

File: src/cmd/namespaces/update/update-namespace.go [hunk 1]
```diff
@@ -9,17 +9,12 @@ import (
 	"github.com/otterize/otterize-cli/src/pkg/utils/prints"
 	"github.com/samber/lo"
 	"github.com/spf13/cobra"
-	"github.com/spf13/viper"
```
Comment (amitlicht) [inline src/cmd/namespaces/update/update-namespace.go:12]: please rename the go file to `associate-namespace-environment.go`

File: src/cmd/namespaces/update/update-namespace.go [hunk 2]
```diff
@@ -30,12 +25,9 @@ var UpdateNamespaceCmd = &cobra.Command{
 		}
 
 		id := args[0]
-		r, err := c.UpdateNamespaceMutationWithResponse(ctxTimeout,
-			id,
-			cloudapi.UpdateNamespaceMutationJSONRequestBody{
-				EnvironmentId: lo.Ternary(viper.IsSet(EnvironmentIDKey), lo.ToPtr(viper.GetString(EnvironmentIDKey)), nil),
-			},
-		)
+		environmentID := args[1]
```
Comment (amitlicht) [inline src/cmd/namespaces/update/update-namespace.go:28]: I'd keep the `--env-id` flag, in-line with our other update subcommand (e.g. `otterize env update add-label <env-id> --key asd ...`). This keeps our CLI structure clearer - all the single-object-update commands accept a single argument (object id), and flags, some required and some optional. Don't forget to mark the environment id flag as required.

**Review by amitlicht — CHANGES_REQUESTED** [review]

File: src/cmd/namespaces/associatetoenvironment/associate-to-environment.go [hunk 3]
```diff
@@ -47,5 +48,9 @@ var UpdateNamespaceCmd = &cobra.Command{
 }
 
 func init() {
-	UpdateNamespaceCmd.Flags().String(EnvironmentIDKey, "", "environment id")
+	AssociateToEnvironmentCmd.Flags().String(EnvironmentIDKey, "", "new environment id for the namespace")
+	err := AssociateToEnvironmentCmd.MarkFlagRequired(EnvironmentIDKey)
```
Comment (amitlicht) [inline src/cmd/namespaces/associatetoenvironment/associate-to-environment.go:52]: use cobra.CheckErr:
`cobra.CheckErr(AssociateToEnvironmentCmd.MarkFlagRequired(EnvironmentIDKey))`

**Review by amitlicht — APPROVED** [review]
see small comment

**Pull request merged**